### ISSUE TITLE
Scale bar should be rounded to hundreths 

### ISSUE THREAD

**Issue by seanmcallister** [mooreryan/iroki_web#7]
to avoid appearance that changing the size doesn't actually change the scale

**Comment by mooreryan** [comment]
Fixed (ed8c8dda8f09de87cb6f5d26a90f8f7d65083a29).  Let me know if that works for you.

**Comment by seanmcallister** [comment]
It still feels strange changing the size without seeing a change in the scale bar because of rounding. Why don't you set the slider to stop at the actual number? Instead of rounding from where the slider stops.

**Comment by mooreryan** [comment]
I changed it to manually entering the size plus an auto switch. Lmk if that works @seanmcallister .

**Comment by seanmcallister** [comment]
Is the auto size supposed to be the mean branch length? Because it changes when you zoom in and seems to just be the minimum length. Zoomed all the way out, I tried to change the default to a smaller value (0.51 to 0.05), and it didn't change (because the auto was the minimum length of the scale bar). Zooming in, the auto length stayed the same (0.51) until I changed the value manually and then reset the auto button (at which point it was the minimum length for that zoom level = 0.04).
